website/docusaurus.config.js

```diff
@@ -15,6 +15,14 @@ const config = {
     colorMode: {
       respectPrefersColorScheme: true,
     },
+    announcementBar: {
+      id: 'support_us',
```

Comment:
Please change to something relevant "play_store_workaround" for example or omit it to use default